src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -86,8 +88,13 @@ public SpongeMod() {
             SimpleCommandService commandService = new SimpleCommandService(this.game.getPluginManager());
             this.game.getServiceManager().setProvider(this, CommandService.class, commandService);
             ((SpongeEventBus) this.game.getEventManager()).register(this, commandService);
-        } catch (ProviderExistsException e1) {
-            logger.warn("Non-Sponge CommandService already registered: " + e1.getLocalizedMessage());
+        } catch (ProviderExistsException e) {
+            logger.warn("Non-Sponge CommandService already registered: " + e.getLocalizedMessage());
+        }
+        try {
+            game.getServiceManager().setProvider(this, SqlService.class, new SqlServiceImpl());
+        } catch (ProviderExistsException e) {
+            logger.warn("Non-Sponge SqlService already registered: " + e.getLocalizedMessage());
```

Comment:
I don't think it's even possible for another provider to be registered at this point.